dotnet/src/webdriver/Remote/HttpCommandExecutor.cs

```diff
@@ -236,6 +223,7 @@ protected virtual void OnSendingRemoteHttpRequest(SendingRemoteHttpRequestEventA
             }
         }
 
+        [MemberNotNull(nameof(client))]
```

Comment:
`CreateHttpClient()` just assign value to `this.client`, why we need this attribute?

Reply:
Once we enable `#nullable` for the full file, we will mark `this.client` as maybe-null. However, after calling this method, we know for sure that `this.client` is not null. So we can do something like

```csharp
this.CreateHttpClient();
this.client.Send(,,,);
```

knowing that the value is never going to be null.

Currently there is not much use for it, but it will be meaningful in follow-up changes.

Reply:
I understand you. But plz understand the code: this code is incorrect.

Reply:
Removed the nullability annotation